.travis.yml

```diff
@@ -15,22 +15,32 @@ cache:
   - apt
 
 addons:
-   apt:
-      packages: ['zlib1g-dev', 'libbz2-dev', 'libsnappy-dev', 'curl', 'libgflags-dev', 'mingw-w64']
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - curl
+      - g++-7
```

Comment:
can we existing tests compile with gcc4.8, and add a build to compile with gcc7? And instead of gcc7, shall we do gcc8?

Reply:
We want to keep rocksdb compatible with gcc4.8. How about we just add one job to compile with gcc7?

Reply:
Done. That job still fails, though.